emcc

```diff
@@ -750,6 +755,16 @@ try:
   # counter for the next index that should be used.
   next_arg_index = len(newargs)
 
+  if pthreads:
+    settings_changes.append('USE_PTHREADS=1')
+    if not any(s.startswith('PTHREAD_POOL_SIZE=') for s in settings_changes):
+      settings_changes.append('PTHREAD_POOL_SIZE=0')
+    js_libraries.append(shared.path_from_root('src', 'library_pthread.js'))
+  else:
+    js_libraries.append(shared.path_from_root('src', 'library_pthread_stub.js'))
```

Comment:
We have in general tried to make it easy for people to just use the same flags during compilation and during linking, I'd prefer to not deviate from that here. Perhaps just making either of those flags be enough is fine (i.e. they both do the same thing).